src/einsteinpy/metric/kerr.py

```diff
@@ -194,13 +193,16 @@ def calculate_trajectory(
             **OdeMethodKwargs
         )
         _scr = self.schwarzschild_r.value * 1.001
+        _event_hor = (
+            kerr_utils.event_horizon(self.schwarzschild_r.value, self.a)[0] * 1.001
+        )
         while ODE.t < end_lambda:
             vec_list.append(ODE.y)
             lambda_list.append(ODE.t)
             ODE.step()
-            if (not singularity_reached) and (ODE.y[1] <= _scr):
+            if (not singularity_reached) and (ODE.y[1] <= _event_hor):
                 warnings.warn(
-                    "r component of position vector reached Schwarzchild Radius. ",
+                    "r component of position vector reached Kerr Radius. ",
```

Comment:
One more change,
Could you change this to `r component of position vector reached event horizon.  `